tinymce/static/django_tinymce/init_tinymce.js

```diff
@@ -35,4 +35,4 @@
       }, 0);
     }, true);
   });
-}(django.jQuery));
+}( function (django) { return django && django.jQuery || jQuery }()));
```

Comment:
You should probably pass the django as a parameter.

    function (django) { return django && django.jQuery || jQuery }(django)